packages/web-pkg/src/helpers/statusIndicators.ts

```diff
@@ -115,6 +115,30 @@ const getProcessingIndicator = ({ resource }: { resource: Resource }): ResourceI
   }
 }
 
+const getSpaceEnabledIndicator = ({ resource }: { resource: Resource }): ResourceIndicator => {
+  return {
+    id: `resource-space-enabled-${resource.getDomSelector()}`,
+    accessibleDescription: $gettext('Space is enabled'),
+    label: $gettext('This space is enabled'),
+    icon: 'play-circle',
+    category: 'system',
+    type: 'resource-space-enabled',
+    fillType: 'line'
+  }
+}
+
+const getSpaceDisabledIndicator = ({ resource }: { resource: Resource }): ResourceIndicator => {
+  return {
+    id: `resource-space-disabled-${resource.getDomSelector()}`,
+    accessibleDescription: $gettext('Space is disabled'),
+    label: $gettext('This space is disabled'),
+    icon: 'stop-circle',
+    category: 'system',
+    type: 'resource-space-enabled',
```

Comment:
```suggestion
    type: 'resource-space-disabled',
```

Reply:
@kulmann is the type correct? This space is disabled but maybe the context is correct?

Reply:
Should be `disabled`, because it's not a broad category but more like an extension-id (= unique for the indicator, but multiple can exist for the multiple rows of the table). I get the feeling that we should move back to the extension system with the indicators.